### PR TITLE
[feature] Allow access to full REST API

### PR DESCRIPTION
**Pull request by GregLeBarbar**
This PR allows to access the full rest API of a wordpress site when user is logged in and is admin OR that the query is from within the cluser (e.g. http://httpd-int:8443/wp-json).

Notes:
  * there are few new endpoints in the wp-json API (see https://make.wordpress.org/core/2020/07/16/new-and-modified-rest-api-endpoints-in-wordpress-5-5/)
  * authentification on rest API: see https://wordpress.stackexchange.com/a/374700 and the relevant plugin file (https://github.com/WP-API/Basic-Auth/blob/master/basic-auth.php)

In summary, these modifications will allow to add new metrics, i.e. plugins usage, in the [wpprober](https://github.com/epfl-si/wp-ops/tree/master/docker/monitoring-wpprobe).

### CODE REVIEW

**Review by ponsfrilus — APPROVED** [review]
pair-programmed
